src/Tribe/iCal.php

```diff
@@ -75,6 +75,19 @@ public function get_ical_link( $type = 'home' ) {
 		return add_query_arg( [ 'ical' => 1 ], $tec->getLink( $type ) );
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$event_date" missing

<sub>hash: 584ab177561b6ee40ede56139d7e135c</sub>